.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -0,0 +1,158 @@
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Other info / logs
+      description: |
+        Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
+    validations:
+      required: false
+  - type: checkboxes
+    id: component
+    attributes:
+      label: What component(s) does this bug affect?
+      description: Please choose one or more components below.
+      options:
+        - label: |
+            `area/artifacts`: Artifact stores and artifact logging
+          required: false
+        - label: |
+            `area/build`: Build and test infrastructure for MLflow
+          required: false
+        - label: |
+            `area/docs`: MLflow documentation pages
+          required: false
+        - label: |
+            `area/examples`: Example code
+          required: false
+        - label: |
+            `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
+          required: false
+        - label: |
+            `area/models`: MLmodel format, model serialization/deserialization, flavors
+          required: false
+        - label: |
+            `area/projects`: MLproject format, project running backends
+          required: false
+        - label: |
+            `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
+          required: false
+        - label: |
+            `area/server-infra`: MLflow Tracking server backend
+          required: false
+        - label: |
+            `area/tracking`: Tracking Service, tracking client APIs, autologging
+          required: false
+  - type: checkboxes
+    id: interface
+    attributes:
+      description: Please choose one or more interfaces below.
+      label: What interface(s) does this bug affect?
+      options:
+        - label: |
+            `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
+          required: false
+        - label: |
+            `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
+          required: false
+        - label: |
+            `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
+          required: false
+        - label: |
+            `area/windows`: Windows support
+          required: false
+  - type: checkboxes
+    id: language
+    attributes:
+      description: Please choose one or more languages below.
+      label: What language(s) does this bug affect?
+      options:
+        - label: |
+            `language/r`: R APIs and clients
+          required: false
+        - label: |
+            `language/java`: Java APIs and clients
+          required: false
+        - label: |
+            `language/new`: Proposals for new client languages
```

Comment:
It appears `|` inserts a newline after a checkbox:

<img width="865" alt="image" src="https://user-images.githubusercontent.com/17039389/165882730-af388fa2-d278-47c6-9707-7aa63633d87c.png">

Source:

```yml
  - type: checkboxes
    id: language
    attributes:
      description: Please choose one or more languages below.
      label: What language(s) does this bug affect?
      options:
        - label: |
            `language/r`: R APIs and clients
          required: false
        - label: |
            `language/java`: Java APIs and clients
          required: false
        - label: |
            `language/new`: Proposals for new client languages
          required: false
  - type: checkboxes
    id: integration
    attributes:
      description: Please choose one or more cloud integrations below.
      label: What integration(s) does this bug affect?
      options:
        - label: "`integrations/azure`: Azure and Azure ML integrations"
          required: false
        - label: "`integrations/sagemaker`: SageMaker integrations"
          required: false
        - label: "`integrations/databricks`: Databricks integrations"
          required: false
```